package.json

```diff
@@ -79,6 +79,9 @@
     "react-redux": "4.4.5",
     "react-router": "2.3.0",
     "react-router-redux": "4.0.4",
+    "react-transform-catch-errors": "1.0.2",
+    "react-transform-hmr": "1.0.4",
+    "redbox-react": "1.2.3",
```

Comment:
I'm not sure I like this one: I'm just not so keen on in-band debug output and personally prefer it out-of-band, i.e. in the console.